calculate.py

```diff
@@ -206,7 +206,24 @@ def check_nmea():
 								STW=''
 								AWS=''
 								AWA=''
-
+							
+							#generate Rate of Turn (ROT)
+							if data_conf.get('STARTUP', 'nmea_rot')=='1' and HDT:
```

Comment:
I am afraid HDT is not defined anywhere. What do you need to create ROT, magnetic heading, true heading or any of them?